utilities/pipelines/e2eValidation/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -269,6 +269,7 @@ function Invoke-ResourceRemoval {
             if ($subscriptionState -eq 'Enabled') {
                 Write-Verbose ('[*] Disabling resource [{0}] of type [{1}]' -f $subscriptionName, $Type) -Verbose
                 if ($PSCmdlet.ShouldProcess("Subscription [$subscriptionName]", 'Remove')) {
+                    Import-Module Az.Accounts -MinimumVersion 4.0.1
```

Comment:
Is this still required here?

Reply:
Yes, it only worked when I specified this here, I can remove it once the agents use the latest version of this module by default